src/styles/rewiews.scss

```diff
@@ -0,0 +1,16 @@
+.rewiew {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $boldTextColor;
+  width: $defaultWidth;
```

Comment:
Ensure that the variables $boldTextColor and $defaultWidth are defined in your SCSS files. If they are not defined, it will cause a compilation error.